.rubocop.yml

```diff
@@ -19,3 +19,5 @@ AllCops:
     - config/**/*
     - bin/**/*
     - Guardfile
+Metrics/BlockLength:
+  Max: 50
```

Comment:
This is a little too much 👀 Maybe we can ignore this cop only for spec files.
If I remember correctly, the syntax to do that is
```yaml
Metrics/BlockLength:
  Exclude:
    - spec/**/*
```